packages/@uppy/utils/types/index.d.ts

```diff
@@ -10,6 +10,8 @@ declare module '@uppy/utils/lib/Translator' {
 
   class Translator {
     constructor (opts: Translator.Locale | Translator.Locale[])
+    translate (key: string, options: object): string
+    translateArray (key: string, options: object): Array<String>
```

Comment:
@goto-bus-stop does it return `Array<String>`? They are JSX elements (so objects?) sometimes, right?

Reply:
Right, the type can be _anything_ that was specified in the `options` object. Without complicating this a lot with generics, the most accurate thing might be to use `any[]` for the return type.